src/Components/App.js

```diff
@@ -1,34 +1,45 @@
 import React, { Component } from 'react'
 import '../CSS_styling/App.css';
-import movieData from '../movie-data';
 import Movies from './Movies';
 import NavBar from './NavBar';
 import Details from './Details'
+import { fetchMovies, getSingleMovie } from '../apiCalls';
 
 class App extends Component {
   constructor() {
    super()
    this.state = {
-     movieInfo: movieData.movies
+     movieInfo: [],
+     error: ''
    }
-   console.log("this.state.movieInfo here---", this.state.movieInfo)
+  }
+
+  componentDidMount = () => {
```

Comment:
We did great handling different errors with a error handling method AND the error boundaries.